Documentation/guides/reading_can_msgs.rst

```diff
@@ -2,61 +2,89 @@
 Reading CAN Messages
 ====================
 
-.. warning:: 
-    Migrated from: 
-    https://cwiki.apache.org/confluence/display/NUTTX/Reading+CAN+Messages
+NuttX's CAN driver's default behavior is to return multiple messages for single
+``read`` operation, if they fit. If your code (especially if migrated from
+SocketCAN) doesn't count with it, then you will most likely encounter seemingly
+lost frames. You have two options: Either implement your code to support his
```

Comment:
s/his/this

Reply:
@Cynerd please fix this typo to let us to merge it